cmd/meroxa/root/resources/create.go

```diff
@@ -51,6 +51,7 @@ type Create struct {
 		ClientCert string `long:"client-cert" short:"" usage:"client certificate for authenticating to the resource"`
 		ClientKey  string `long:"client-key"  short:"" usage:"client private key for authenticating to the resource"`
 		SSL        bool   `long:"ssl"         short:"" usage:"use SSL"`
+		SSHURL     string `long:"ssh-url"     short:"" usage:"SSH tunneling address"`
```

Comment:
Shouldn't this flag be hidden until we make this publicly available?

Reply:
Confirmed with products that we are okay with leaving ssh tunnel in the UI for now: https://meroxa.slack.com/archives/C01JL1D0V29/p1623100112056200?thread_ts=1623087396.051400&cid=C01JL1D0V29